integration-test/src/test/scala/org/apache/pekko/persistence/jdbc/integration/MySQLDurableStateStorePluginSpec.scala

```diff
@@ -0,0 +1,21 @@
+/*
```

Comment:
new files should use the standard Apache header unless there are methods copied in from existing files - where we should use that file's header

eg https://github.com/apache/pekko-persistence-jdbc/blob/main/project/PekkoCoreDependency.scala#L1